modules/core/src/main/java/org/apache/ignite/logger/java/JavaLoggerFormatter.java

```diff
@@ -19,47 +19,72 @@
 import java.io.PrintWriter;
 import java.io.StringWriter;
 import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
 import java.util.logging.Formatter;
 import java.util.logging.LogRecord;
+import org.apache.ignite.IgniteSystemProperties;
 import org.apache.ignite.internal.util.IgniteUtils;
 import org.apache.ignite.internal.util.typedef.internal.S;
+import org.jetbrains.annotations.NotNull;
 
 /**
  * Formatter for JUL logger.
  */
 public class JavaLoggerFormatter extends Formatter {
     /** Name for anonymous loggers. */
-    public static final String ANONYMOUS_LOGGER_NAME = "UNKNOWN";
+    static final String ANONYMOUS_LOGGER_NAME = "UNKNOWN";
+
+    /** Default date-time format. */
+    static final DateTimeFormatter DEFAULT_DATE_FMT = IgniteUtils.ISO_DATE_FMT;
+
+    /** Log message date-time formatter. */
+    static final DateTimeFormatter DATE_FMT = resolveDateFormat();
+
+    /** Property for overriding default dat-time format. */
+    static final String IGNITE_JUL_DATE_FORMAT_PROPERTY = "IGNITE_JAVA_LOGGER_DATE_FORMAT";
```

Comment:
How is this property related to the issue described in the ticket? What problem are we solving here?